gr-zeromq/python/zeromq/qa_zeromq_pubsub.py

```diff
@@ -48,5 +48,28 @@ def test_001 (self):
         self.tb.wait()
         self.assertFloatTuplesAlmostEqual(sink.data(), src_data)
 
+    def test_002(self):
+        # Test getting the port returns a port we explicitly set endpoint to
+        zeromq_pub_sink = zeromq.pub_sink(gr.sizeof_float, 1, "tcp://127.0.0.1:5556", 0)
+        self.assertEqual(5556, zeromq_pub_sink.port())
```

Comment:
I hadn't seen the hard-coding in the other QA tests and agree we shouldn't be doing that. If you want to make that change, make it a new PR though after this one is dealt with.